Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -45,6 +45,11 @@
     intensitySlope: 3
     totalIntensity: 120 # about a ~4 tile radius
     canCreateVacuum: false
+  - type: OnUseTimerTrigger
+    beepSound:
+      path: "/Audio/Effects/double_beep.ogg"
```

Comment:
Can you get a new sound for this? It's already bad enough turrets and health analyzer sound similar.